mbtb_app/web_portal/mbtb_portal/assets/js/suspend_user_form.js

```diff
@@ -0,0 +1,19 @@
+
+  let requests_ids = get_checkbox_values();
+
+  let suspend_reason=document.getElementById("reason").value;
+
+  let post_request = $.post( "/suspend-user-with-reason", {requests_ids: requests_ids}, function(data, status) {
```

Comment:
3 -> Here, post url should be `/suspend_user_form`, this will call the controller (which will load form) and send selected request_ids along with it.